coderd/authzquery/apikey.go

```diff
@@ -0,0 +1,40 @@
+package authzquery
+
+import (
+	"context"
+	"time"
+
+	"github.com/coder/coder/coderd/rbac"
+
+	"github.com/coder/coder/coderd/database"
+)
+
+func (q *AuthzQuerier) DeleteAPIKeyByID(ctx context.Context, id string) error {
+	return authorizedDelete(q.logger, q.authorizer, q.database.GetAPIKeyByID, q.database.DeleteAPIKeyByID)(ctx, id)
```

Comment:
nit: these lines should be broken at something reasonable like 100 chars

Reply:
Oof, my IDE line is set to 120. We do go over this often too though.